gtests/net/mptcp/mp_join/mp_join_client.pkt

```diff
@@ -22,8 +22,11 @@
 +0.0 > > addr[saddr1] S 0:0(0) <mss 1460,sackOK,TS val 448955294 ecr 0,nop,wscale 8,mp_join_syn address_id=0 token=sha256_32(skey)>
 +0.0 < S. 0:0(0) ack 1 win 65535 <mss 1460,sackOK,TS val 448955294 ecr 448955294,nop,wscale 8,mp_join_syn_ack address_id=1 sender_hmac=auto>
 +0.0 > . 1:1(0) ack 1 <nop,nop,TS val 448955294 ecr 448955294,mp_join_ack sender_hmac=auto>
-+0.0 < . 1:1(0) ack 1 win 256 <nop,nop,TS val 448955294 ecr 448955294>
++0.0 < . 1:1(0) ack 1 win 256 <nop,nop,TS val 448955294 ecr 448955294, dss dack8=3 nocs>
++0.0 < . 1:101(100) ack 1 win 256 <TS val 448955294 ecr 448955294, dss dack8=3 dsn8=1 ssn=1 dll=100 nocs>
++0.0 > . 1:1(0) ack 101 win 256 <nop, nop, TS val 448955294 ecr 448955294, add_address addr[saddr1], dss dack8=101 nocs>
```

Comment:
is it the echo bit? I mean: I see there is no hmac but I guess Packetdrill doesn't compare the echo bit, does it?

Reply:
correct, if you specify address but not hmac, then the packetdrill script parser assumes that it's an echoed ADD_ADDR. Oh, and theoretically that pure inbound DACK (1:1(0)) and the 100 byte  segment (1:101(100)) can also collapse into a single packet

Reply:
Thank you for the reply.

> Oh, and theoretically that pure inbound DACK (1:1(0)) and the 100 byte segment (1:101(100)) can also collapse into a single packet

Indeed, we could do that. But I guess we can also do this improvement later :-)

I would suggest here to merge this to fix issues and check the rest later.